code/modules/reagents/chemistry/recipes/slime_extracts.dm

```diff
@@ -477,17 +477,6 @@
 	..()
 
 //Adamantine
-/datum/chemical_reaction/slime/slimegolem
-	name = "Slime Golem"
-	id = "m_golem"
-	required_reagents = list("plasma" = 1)
-	required_container = /obj/item/slime_extract/adamantine
-	required_other = 1
-
-/datum/chemical_reaction/slime/slimegolem/on_reaction(datum/reagents/holder)
-	new /obj/effect/golemrune(get_turf(holder.my_atom))
-	..()
-
 /datum/chemical_reaction/slime/slimegolem2
 	name = "Slime Golem 2"
```

Comment:
also correct the names/types that refer to it as slimegolem2 now that it's the only one.